subsys/bluetooth/host/iso.c

```diff
@@ -693,6 +693,35 @@ static bool valid_chan_io_qos(const struct bt_iso_chan_io_qos *io_qos,
 }
 
 #if defined(CONFIG_BT_ISO_UNICAST)
+static int bt_iso_accept(struct bt_conn *acl, struct bt_conn *iso)
```

Comment:
Mixed `bt_iso_` and `hci_le_`  namespaces used for global and file static symbols. Looks like host code widely uses mixed namespaces in modules for public, global, file static and private symbols.

```suggestion
static int accept(struct bt_conn *acl, struct bt_conn *iso)
```

Reply:
It does indeed, but afaik we don't have any rules, or even recommendations regarding this, do we? 

@joerchan @jhedberg Do you have any opinions on whether we should omit `bt_xx_` for static functions?

Reply:
I’ve usually omitted the topmost namespace to give a hint that the function is not public. In this case the name would then be iso_accept()

Reply:
I'm more leaning towards `iso_accept` if we really want to change this.